### PR TITLE
[EOS-16739] [Raised on behalf of Kapil Jinna] Counting objects in an index

### PR DESCRIPTION
**Pull request by SeagateChaDeepak**
1. Added a class to count objects in an index.
2. Wrote a script that call the wrapper above to list object in an index.

http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server-7.8.2003/567/

### CODE REVIEW

File: scripts/count_kv.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3.6
+
+import sys
+import os
```
Comment (vimalraghwani20) [inline scripts/count_kv.py:4]: remove unused import as suggested by codacy.

**Review by vimalraghwani20 — APPROVED** [review]
+1

File: scripts/count_kv.py (hunk 1 above)
Comment (SeagateChaDeepak) [reply scripts/count_kv.py]: Done.

**Review by nileshgovande — APPROVED** [review]

File: scripts/count_kv.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3.6
```
Comment (nileshgovande) [inline scripts/count_kv.py:1]: File header missing

**Comment by nileshgovande** [comment]
Pre-merge #74 has passed

**Pull request merged**